boefjes/boefjes/job_handler.py

```diff
@@ -207,16 +229,31 @@ def handle(self, normalizer_meta: NormalizerMeta) -> None:
                         valid_time=normalizer_meta.raw_data.boefje_meta.ended_at,
                     )
                 )
+
+            validated_scan_profiles = [
+                profile
+                for profile in results.scan_profiles
+                if self.whitelist and profile.level <= self.whitelist.get(normalizer_meta.normalizer.id, -1)
```

Comment:
Maybe we should just clamp the requested scan-level to the max allowed scan level instead of ignoring them. 

This could be as simple as:
` profile.level  = min(self.whitelist.get(normalizer_meta.normalizer.id, -1), profile.level)`
if whitelisted, and if it returns higher than -1 we can proceed.

Reply:
Not sure what makes the most sense, but for now we could definitely just do this